src/ucp/core/ucp_request.inl

```diff
@@ -264,6 +264,8 @@ ucp_request_complete_tag_recv(ucp_request_t *req, ucs_status_t status)
                   req->recv.tag.info.sender_tag, req->recv.tag.info.length,
                   ucs_status_string(status));
     UCS_PROFILE_REQUEST_EVENT(req, "complete_tag_recv", status);
+    /* coverity[address_free] */
```

Comment:
for which field are those two comments for?

Reply:
there are bogus Coverity errors that calling a completion callback may release the request (because coverity thinks the completion callback can resolve to some internal function such as ucp_cm_server_conn_request_progress